client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -279,3 +287,12 @@ module.exports = React.createClass( {
 	}
 
 } );
+
+export default connect(
+	( state, props ) => {
```

Comment:
I get a linting warning here because props is defined but never used. You might want to run ESLint against these files, just in case. :)